### PR TITLE
The place ship for the hard level of difficulty

### PR DESCRIPTION
**Pull request by lehmana**
This will change where the ships are placed when the game is on hard level of difficulty. The model auto places the ships hard coded (the easy mode) but if the hard is selected then it moves them to new random locations

### CODE REVIEW

**Comment by lehmana** [comment]
oh and i deleted some dead code to do a mini amount of refactoring

**Comment by jahneff** [comment]
Tested and it works great! And I like your descriptive function name.

File: src/main/resources/public/js/app.js [hunk 1]
```diff
@@ -20,6 +20,8 @@ $( document ).ready(function() {
                 //The default difficulty is Easy
                 if ( isHardDifficulty) {
                     gameModel.hardAI = 1;
+                    //the following will call a function that will change the computer's ship placement to be random
```
Comment (sharipol) [inline src/main/resources/public/js/app.js:23]: Awesome name choice for the function, it's very descriptive

File: src/main/resources/public/js/app.js [hunk 2]
```diff
@@ -38,8 +40,6 @@ $( document ).ready(function() {
 
 function placeShip(x,y) {
    console.log($( "#shipSelec" ).val());
-//   console.log($( "#rowSelec" ).val());
```
Comment (sharipol) [inline src/main/resources/public/js/app.js:41]: Good job on deleting dead code

File: src/main/resources/public/js/app.js [hunk 3]
```diff
@@ -102,9 +97,6 @@ function fire(x, y){
 }
 
 function scan(x,y){
- //console.log($( "#colScan" ).val());
-   //console.log($( "#rowScan" ).val());
-//var menuId = $( "ul.nav" ).first().attr( "id" );
```
Comment (sharipol) [inline src/main/resources/public/js/app.js:107]: So much dead code getting cleared!

Comment (sharipol) [reply src/main/resources/public/js/app.js:107]: Great job!

File: src/main/resources/public/js/app.js [hunk 4]
```diff
@@ -211,7 +202,102 @@ function displayShip(ship){
         }
     }
  }
+}
 
-
-
+function changeComputerShipPlacement() {
+    //This is the function that will randomize the placement of the computers ships when it is on difficulty level hard
+    //this master number is the number that will determine if we are dealing with horizontal ships or vertical ships
```
Comment (sharipol) [inline src/main/resources/public/js/app.js:209]: Good job on commenting what each part of the function does

**Review by sharipol — COMMENTED** [review]
Great code! Very descriptive function name and good commenting throughout the code, also nice job on deleting all that dead code.

**Pull request merged**